requirements/pip.txt

```diff
@@ -1,11 +1,11 @@
 # Base packages
 
-pip==20.2.1
-virtualenv==20.0.30
+pip==20.2.2
+virtualenv==20.0.31
 
 django==2.2.15  # pyup: <2.3
 django-extensions==3.0.5
-django_polymorphic==2.1.2
+django_polymorphic==3.0.0
```

Comment:
Looks like the mayor change was dropping support for python2 https://github.com/django-polymorphic/django-polymorphic/blob/master/docs/changelog.rst#changes-in-300-2020-08-21